packages/components/field-label/src/field-label.visualroute.jsx

```diff
@@ -3,10 +3,16 @@ import {
   WarningIcon,
   FlatButton,
 } from '@commercetools-frontend/ui-kit';
+import styled from '@emotion/styled';
 import { Suite, Spec } from '../../../../test/percy';
 
 export const routePath = '/field-label';
 
+const BorderedBox = styled.div`
```

Comment:
These are components to help with the new VRT where we render the `FieldLabel` with a description which is a component rendering nothing to make sure the unexpected vertical spacing does not exist anymore.